### PR TITLE
docs: custom-vnet.md sanitation 

### LINKED ISSUE #561: docs: Missing notes about "clusterSubnet" in custom-vnet.md
**Is this a request for help?**:
 no
---

**Is this an ISSUE or FEATURE REQUEST?** (choose one):
 issue
---

**What version of aks-engine?**:
 latest
---

**Kubernetes version**:
1.12.5

**What happened**:
Following tutorial custom-vnet.md to install kubernetes cluster, subnet ip ranges (10.100.0.0/24, 10.200.0.0/24). kube-proxy and kube-controller-manager still use default clusterCIDR 10.124.0.0/12. 
```
kube-controller-manager
    - --cluster-cidr=10.240.0.0/12

kube-proxy
    --cluster-cidr=10.240.0.0/12
```
This causes packet from pod to service ip get masqueraded and break network policy. 
Adding `clusterSubnet` to kubernetesConfig fixed the issue.

**What you expected to happen**:
A note in tutorial about setting up clusterCIDR. 

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by ams0**
**Reason for Change**:
A minor doc change to remove references to Swarm. Also, in the case of AzureCNI, `--cluster-cidr` values are ignored and pods have connectivity to Service IPs (as the issue reference below seems to suggest, see comments there).

**Issue Fixed**:

Fixes #561 

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [X] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: docs/tutorials/custom-vnet.md [hunk 1]
```diff
@@ -3,9 +3,7 @@
 In this tutorial you are going to learn how to use [AKS Engine](https://github.com/Azure/aks-engine) to deploy a brand new cluster into an existing or pre-created virtual network.
 By doing this, you will be able to control the properties of the virtual network or integrate a new cluster into your existing infrastructure.
 
-*Note: This article describes the procedure with Docker Swarm but it will work in the exact same way with the all the orchestrators available with AKS Engine: Docker Swarm, Kubernetes and DC/OS.*
-
-*For Kubernetes, the cluster should be deployed in the same resource group as the virtual network and the service principal you use for the cluster needs permissions on the VNET resource's group too. Custom VNET for Kubernetes Windows cluster has a [known issue](https://github.com/Azure/aks-engine/issues/371).*
+_For Kubernetes, the cluster should be deployed in the same resource group as the virtual network and the service principal you use for the cluster needs permissions on the VNET resource's group too._
```
Comment (jackfrancis) [inline docs/tutorials/custom-vnet.md]: We can probably just remove the "For Kubernetes" part and just begin the sentence like "The Kubernetes cluster should be deployed in the same resource group..."

File: docs/tutorials/custom-vnet.md [hunk 2]
```diff
@@ -89,40 +92,28 @@ Depending on the orchestrator you want to deploy, the number of agent pools, the
 
 There are a lot of examples available on the [AKS Engine GitHub](https://github.com/Azure/aks-engine/tree/master/examples) and you can find [one dedicated for virtual network](https://github.com/Azure/aks-engine/blob/master/examples/vnet/README.md).
 
-In this case, we are going to use the following template:
+In this case, we are going to use the following template (this creates a cluster with AzureCNI network plugin):
```
Comment (jackfrancis) [inline docs/tutorials/custom-vnet.md]: s/AzureCNI/Azure CNI/ (Add a space)

**Comment by jackfrancis** [comment]
Thank you @ams0!

File: docs/tutorials/custom-vnet.md (hunk 2 above)
Comment (mboersma) [reply docs/tutorials/custom-vnet.md]: I would also say "**the** Azure CNI network plugin".

File: docs/tutorials/custom-vnet.md (hunk 1 above)
Comment (ams0) [reply docs/tutorials/custom-vnet.md]: addressed in https://github.com/Azure/aks-engine/pull/1282/commits/1b31865c36284c0981ad146e7a9dd7001690bd7a

File: docs/tutorials/custom-vnet.md (hunk 2 above)
Comment (ams0) [reply docs/tutorials/custom-vnet.md]: addressed in https://github.com/Azure/aks-engine/pull/1282/commits/1b31865c36284c0981ad146e7a9dd7001690bd7a

File: docs/tutorials/custom-vnet.md [hunk 3]
```diff
@@ -134,29 +125,33 @@ In this case, we are going to use the following template:
...
-*Note: Make sure the the vnetSubnetId matches with your subnet, by giving your **SUBSCRIPTION_ID**, **RESOURCE_GROUP_NAME**, virtual network and subnet names. You also need to fill DNS prefix for all the public pools you want to create, give an SSH keys...*
+_Note: Make sure the the vnetSubnetId matches with your subnet, by giving your **SUBSCRIPTION_ID**, **RESOURCE_GROUP_NAME**, virtual network and subnet names. You also need to fill DNS prefix for all the public pools you want to create, give an SSH keys..._
 
 ## Generate the cluster Azure Resource Manager template
 
 Once your are ready with the cluster definition file, you can use AKS Engine to generate the ARM template that will be used to deploy the cluster on Azure:
 
 ```bash
-aks-engine azuredeploy.swarm.clusterdefinition.json
+aks-engine custom-vnet.json
```
Comment (CecileRobertMichon) [inline docs/tutorials/custom-vnet.md]: isn't this missing the `generate` verb?

Comment (CecileRobertMichon) [reply docs/tutorials/custom-vnet.md]:
```suggestion
aks-engine generate custom-vnet.json
```

**Comment by CecileRobertMichon** [comment]
closing in favor of #1399

**Pull request closed without merging**